typings/index.d.ts

```diff
@@ -3784,9 +3784,10 @@ export interface HTTPOptions {
   headers?: Record<string, string>;
 }
 
-export interface ImageURLOptions extends StaticImageURLOptions {
+export type ImageURLOptions = StaticImageURLOptions & {
```

Comment:
```suggestion
export interface ImageURLOptions extends StaticImageURLOptions {
```

Reply:
Or if it complains (although not suggested, as it makes them non-assignable to each other):

```suggestion
export interface ImageURLOptions extends Omit<StaticImageURLOptions, 'format'> {
```

Reply:
I used the former initially, but I then switched to a `type` on the second commit because it was complaining